lib/oli/delivery/metrics.ex

```diff
@@ -985,19 +985,19 @@ defmodule Oli.Delivery.Metrics do
             summary.user_id != -1 and
             summary.resource_type_id == ^page_type_id,
         where: ^filter_by_container,
-        group_by: summary.user_id,
+        group_by: [summary.user_id, summary.num_first_attempts],
```

Comment:
`summary.num_first_attempts` is required to be in the group_by clause in order to be in the select clause

Reply:
Changed to use `sum` aggregation instead, this also fixed the intermittently broken unit test and is the more correct approach